### ISSUE TITLE
Modus NavBar - Other elements are overlapping the User Menu

### ISSUE THREAD

**Issue by ElavarasanTrimble** [trimble-oss/modus-web-components#1730]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-web-components/blob/main/CONTRIBUTING.md)

### Describe the issue

When Opening User Menu the Element below the NavBar is overlapping the User Menu.

For Example, the <h2> is overlapping the User Menu.

![Screenshot 2023-09-25 161431](https://github.com/trimble-oss/modus-web-components/assets/102205488/efe74b23-364a-4128-85de-fb1b0cb29b98)
![Screenshot 2023-09-25 161407](https://github.com/trimble-oss/modus-web-components/assets/102205488/36e1f4d5-ce64-40ac-89c5-ec142984b7bc)

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What is the issue regarding ?

@trimble-oss/modus-web-components

### Priority

Medium

### What product/project are you using Modus Components for ?

e-Builder

### What is your team/division name ?

e-Builder

### Are you willing to contribute ?

None

**Comment by msankaran0712** [comment]
@ElavarasanTrimble Is it fixable by increasing the z-index of other elements on the page?

**Comment by ElavarasanTrimble** [comment]
@msankaran0712 yes, but our concern is we would have to potentially change every page in the app?

**Comment by msankaran0712** [comment]
@ElavarasanTrimble, I see we need a z-index configuration for the Profile menu pop-up. Are you interested in contributing?

**Comment by cjwinsor** [comment]
Update Profile Menu z-index to be lower than modal's default.  Do not introduce a z-index property at this time, we will try to manage ourselves.  I can also update e-builders z-index to be reasonably lower than our defaults.

**Comment by prashanth-offcl** [comment]
@ElavarasanTrimble / @cjwinsor Any update on this?

FYI: Currently, the default modus modal zIndex is set to 1 and it is configurable via zIndex prop, the navbar menu zIndex is 100.
